client/src/main/proto/spine/base/command.proto

```diff
@@ -79,8 +79,12 @@ message CommandContext {
     // The `namespace` attribute must be defined for commands in multitenant applications.
     Namespace namespace = 5;
 
+    // An identifier of a software component, which generated the command.
+    // If the source is a Bounded Context, this field is its name.
+    string source = 6;
```

Comment:
This information may be useful, but it would be better to have it as a client environment definition, which may contain: client type, OS, code version etc.